src/data/redux.test.jsx

```diff
@@ -711,173 +994,16 @@ describe('Data layer integration tests', () => {
-      const examURL = `${getConfig().EXAMS_BASE_URL}/api/v1/student/exam/attempt/course_id/${courseId}/content_id/${contentId}`;
-      const activeAttemptURL = `${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`;
-
-      axiosMock.onGet(examURL).reply(200, { exam });
-      axiosMock.onGet(activeAttemptURL).reply(200, { attempt });
-      axiosMock.onPut(updateExamAttemptURL).reply(200, { exam_attempt_id: attempt.id });
-
-      await executeThunk(thunks.getExamAttemptsData(courseId, contentId), store.dispatch);
-      await executeThunk(thunks.stopExam(), store.dispatch, store.getState);
-      expect(axiosMock.history.put[0].url)
-        .toEqual(updateExamAttemptURL);
-    });
-
-    it('Should call the exams service to fetch attempt data', async () => {
-      const examURL = `${getConfig().EXAMS_BASE_URL}/api/v1/student/exam/attempt/course_id/${courseId}/content_id/${contentId}`;
-      const activeAttemptURL = `${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`;
-
-      axiosMock.onGet(examURL).reply(200, { exam });
-      axiosMock.onGet(activeAttemptURL).reply(200, attempt);
-
-      await executeThunk(thunks.getExamAttemptsData(courseId, contentId), store.dispatch);
-
-      expect(axiosMock.history.get[0].url)
-        .toEqual(examURL);
-      expect(axiosMock.history.get[1].url)
-        .toEqual(activeAttemptURL);
-
-      const state = store.getState();
-      expect(state)
-        .toMatchSnapshot();
-    });
-
-    it('Should call the exams service to get latest attempt data', async () => {
-      const activeAttemptURL = `${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`;
-
-      // Updated attempt with changed status
-      const updatedAttempt = Factory.build('attempt', { attempt_status: ExamStatus.READY_TO_SUBMIT });
-
-      // Get initial data first, then updated data when calling pollAttempt
-      axiosMock.onGet(activeAttemptURL).replyOnce(200, attempt);
-      axiosMock.onGet(activeAttemptURL).reply(200, updatedAttempt);
-
-      // Get data, initialize state
-      await executeThunk(thunks.getLatestAttemptData(courseId), store.dispatch);
-      const beforeState = store.getState();
-      expect(beforeState.examState.activeAttempt).toEqual(attempt);
-
-      // Poll with initialized state
-      const dummyURL = `${getConfig().EXAMS_BASE_URL}/edx-proctoring/dummy-url`;
-      await executeThunk(thunks.pollAttempt(dummyURL), store.dispatch, store.getState);
-      const afterState = store.getState();
-      expect(afterState.examState.activeAttempt).toEqual(updatedAttempt);
-
-      expect(axiosMock.history.get[0].url).toEqual(activeAttemptURL);
-      expect(axiosMock.history.get[1].url).toEqual(activeAttemptURL);
-
-      expect(afterState).toMatchSnapshot();
-      expect(beforeState).not.toEqual(afterState); // Test that the state was updated when polled
-    });
-  });
-});
-
-describe('External API integration tests', () => {
```

Comment:
moved these to a different file since they don't have to do with the redux store and this file is already ginormous